tests/unit/cartography/intel/github/test_teams.py

```diff
@@ -134,7 +134,7 @@ def test_get_team_repos_github_returns_none(mock_sleep, mock_get_team_repos):
     mock_get_team_repos.return_value = mock_team_repos
 
     # Assert we raise an exception
-    with pytest.raises(RuntimeError):
+    with pytest.raises(TypeError):
```

Comment:
TypeError here rather than RuntimeError because in the old code, it explicitly threw a RuntimeError, whereas in the new code, it throws whatever the underlying error is, which in this case is TypeError.